features/docs/formatters/event_stream_formatter.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Event Stream Formatter
+
+  Emits a stream of events compatible with https://github.com/cucumber/cucumber-pretty-formatter
+
+  Background:
+    Given the standard step definitions
+
+  Scenario: A simple scenario
+    Given a file named "features/simple.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given this step passes
+      """
+    When I run `cucumber -q --format Cucumber::Formatter::EventStream`
+    Then it should pass
+    And the stdout should contain exactly:
+      """
+      {"event":"TestRunStarted","protocol_version":"0.0.1"}
+      {"event":"GherkinSourceRead","id":"features/simple.feature:1","source":"Feature:\n  Scenario:\n    Given this step passes"}
+      {"event":"StepDefinitionFound","source_id":"features/simple.feature:3","definition_id": "features/step_definitions/steps.rb:1"}
+      {"event":"TestCaseStarted","id":"features/simple.feature:2"}
+      {"event":"TestStepStarted","id":"features/simple.feature:3"}
+      {"event":"TestStepPassed","id":"features/simple.feature:3"}
+      {"event":"TestCasePassed","id":"features/simple.feature:2"}
```

Comment:
Failed or Undefined events are different because they have more details:

``` json
{"event":"TestCaseUndefined", "summary": "TODO: implement step definition", "snippet": "some src snippet may be provided"}
```

Though, these can optional fields. I agree it could be a single event in this case, because we have few statuses:

- passed
- failed
- skipped
- undefined
- ambiguous
- pending